practices/responsibility.md

```diff
@@ -0,0 +1,91 @@
+---
+slug: responsibility
+stages:
+  - development
+short_description: Responsibility is keeping up the trust and faith of people who trusts you, no matter what the situation is. Responsibility primarily involves two things -- authority and accountability; however, trust is often a factor as well.
```

Comment:
Our plagiarism check found similar text to this. It is on Quora (https://www.quora.com/What-is-responsibility) and it is from Abhimanyu Bellam: _Responsibility is keeping up the trust and faith of people (who have trust on u) no matter what the situation is._

Other similarity can be found with Quora (https://www.quora.com/How-would-you-define-responsibility) text from James Longwell: _Responsibility primarily involves two things: authority and accountability; however, trust is often a factor as well_

Reply:
Hi, 
Thank you so much for reviewing this and suggesting me the changes. First of all, I am extremely sorry for increasing your work and after checking all the suggested changes, I am going to re-write the entire article on my own.
Kindly, give me few days to create the article once again.